.github/workflows/requirements-up-to-date.yml

```diff
@@ -0,0 +1,50 @@
+name: Validate requirements.txt
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  validate-dependencies:
```

Comment:
Functionality taken from @aaronhaefner's work on this PR: https://github.com/uchicago-capp-30320/new-arrivals-chi/pull/159